### PR TITLE
Automatic snapshot commit from seacas at ea676bc

### PR DESCRIPTION
**Pull request by gsjaardema**
Origin repo remote tracking branch: 'origin/master'
Origin repo remote repo URL: 'origin = https://github.com/gsjaardema/seacas'

At commit:

commit e7fb40119a6b912835383179026138048dabaa08
Author:  Greg Sjaardema <gsjaardema@gmail.com>
Date:    Thu May 9 08:35:20 2019 -0600
Summary: IOSS: Eliminate shadowed variable warning [ci skip]

---------------------------------------
* strcpy -> copy_string replacements
* add fmt:: library and use in IOSS (https://fmtlib.net/latest/index.html)
* update hash includes
* add part-prefix option to ejoin to avoid name collision
* update ejoin offset documentation
* convert some strlen==0 to .empty() calls
* convert some char* to std::string in nem_slice/nem_spread
* add test of cgns append to file with block reordering
* improved db summary output in Ioss::Region and io_info
* cleaned up qa output in IOSS applications
* fix cgns/hdf5 hang due to mismatched info output based on nodename
* improved testing of structured decomposition

@trilinos/seacas 

## Checklist

- [x] My commit messages mention the appropriate GitHub issue numbers.
- [x] My code follows the code style of the affected package(s).
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [code contribution guidelines](../blob/master/CONTRIBUTING.md) for this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] No new compiler warnings were introduced.
- [ ] These changes break backwards compatibility.

### CODE REVIEW

File: packages/seacas/applications/ejoin/EJ_Version.h [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
 static char const *qainfo[] = {
     "ejoin",
-    "2018/07/26",
-    "1.5.2",
+    "2019/05/06",
+    "1.5.3",
```
Comment (bartlettroscoe) [inline packages/seacas/applications/ejoin/EJ_Version.h]: How can the date be newer but the version be older?

Comment (gsjaardema) [reply packages/seacas/applications/ejoin/EJ_Version.h]: My math shows both are newer...

Comment (bartlettroscoe) [reply packages/seacas/applications/ejoin/EJ_Version.h]: Must have been an optical illusion. I could have sworn that it was '1.5.3' on the left and '1.5.2' on the right.  Sorry about that.

**Comment by mhoemmen** [comment]
@gsjaardema FYI, the CUDA PR test is currently broken for reasons likely unrelated to Trilinos: https://github.com/trilinos/Trilinos/issues/5147

**Pull request closed without merging**